COLLABORATOR_GUIDE.md

```diff
@@ -520,13 +524,13 @@ The TSC should serve as the final arbiter where required.
    present.
 1. Review the commit message to ensure that it adheres to the guidelines
    outlined in the [contributing][] guide.
-1. Add all necessary [metadata](#metadata) to commit messages before landing.
-   See the commit log for examples such as [this
-   one](https://github.com/nodejs/node/commit/b636ba8186) if unsure exactly how
-   to format your commit messages.
+1. Add all necessary [metadata](#metadata) to commit messages before landing. If
+   You are unsure exactly how to format the commit messages, use the commit log
+   as a reference, for examples commits such as [this one][commit-example].
```

Comment:
I think the sentence should end here at 'reference' and the last part re-worded. Perhaps something like 'See [this commit][commit-example] as an example.'